src/schedule/timer_domain.c

```diff
@@ -42,24 +42,24 @@
 #define ZEPHYR_SCHED_COST	CYC_PER_TICK
 
 K_THREAD_STACK_DEFINE(ll_workq_stack0, ZEPHYR_LL_WORKQ_SIZE);
-#if CONFIG_CORE_COUNT > 1
+#if PLATFORM_CORE_COUNT > 1
```

Comment:
Hmm, why do we have CONFIG_CORE_COUNT and PLATFORM_CORE_COUNT ? They look like the same thing, we should only have one. Could this fix/merge be done as a separate fix as it's confusing and should come from Kconfig.

Reply:
I found a way to achieve the same without using `PLATFORM_CORE_COUNT`, I'll make a separate patch to get rid of it